pages/task/read/views/project.jsx

```diff
@@ -39,21 +40,29 @@ const Project = ({ task, formFields, project, establishment, children }) => {
             </Fragment>
         }
       </StickyNavAnchor>
+
       {
-        !!task.nextSteps.length && (
+        schema.status.options.length > 0 &&
           <StickyNavAnchor id="status">
             <h2><Snippet>sticky-nav.status</Snippet></h2>
             <p><Snippet>make-decision.hint</Snippet></p>
-            {
-              formFields
-            }
+            <Form />
+            { task.canBeWithdrawn && <WithdrawApplication showHeading /> }
+          </StickyNavAnchor>
+      }
+
+      {
+        schema.status.options.length === 0 && task.canBeWithdrawn &&
+          <StickyNavAnchor id="withdraw">
+            <h2><Snippet>sticky-nav.withdraw</Snippet></h2>
+            { task.canBeWithdrawn && <WithdrawApplication /> }
```

Comment:
This extra condition is redundant since it's already been applied 3 lines up ☝️

Reply:
Good point, this is from moving stuff in and out of the conditions many times to try and get the logic right.